public/app/features/explore/Logs/Logs.tsx

```diff
@@ -463,7 +463,7 @@ class UnthemedLogs extends PureComponent<Props, State> {
 
     return {
       from: new Date(this.props.absoluteRange.from).toISOString(),
-      to: new Date(prevLog ? prevLog.timeEpochMs : this.props.absoluteRange.to).toISOString(),
+      to: new Date(prevLog ? new Date(prevLog.timeEpochMs + 1) : this.props.absoluteRange.to).toISOString(),
```

Comment:
`prevLog`, the closest most recent log after the one that we want to share, should include `row` timestamp. E.g.
Timestamps: 5, 4, 3, 2, 1
Visible range: From 1 to 5
If we want to share 4, the time range should be from 1 to 5
If we want to share 2, the time range should be from 1 to 3.

I wonder if there's a scenario that this code is not considering, because it shouldn't be necessary.

Reply:
It seems that `prevLog` should be the previous log that also has a more recent timestamp.

Reply:
Oh, yea, that's also a way to fix it. Let me do it like that.